openscap_report/report_generators/html_templates/js/interactive_script.js

```diff
@@ -128,4 +128,9 @@ const selector = ".pf-c-clipboard-copy__text";
 document.querySelectorAll("#copy").forEach(el => el.addEventListener('click', () => {
     const el_with_text = el.parentElement.parentElement.parentElement.querySelector(selector);
     navigator.clipboard.writeText(el_with_text.textContent);
-}));
\ No newline at end of file
+}));
+
+function clear_input(self) { // eslint-disable-line no-unused-vars
+    self.previousSibling.value= "";
+    search_rule(self.previousSibling);
+}
```

Comment:
Newline?

Reply:
Probably yes, I use eslint to check javascript files and the default is forced without a newline. I've read the documentation for the [`eol-last`](https://eslint.org/docs/latest/rules/eol-last) rule, and it doesn't explain any specific purpose or benefit of not having a newline at the end of the file. So I'll use the Unix conversion and fix the newlines and update the line settings in another PR.